src/Symfony/Component/Config/Loader/FileLoader.php

```diff
@@ -79,6 +82,85 @@ public function getLocator()
      */
     public function import($resource, $type = null, $ignoreErrors = false, $sourceResource = null)
     {
+        $ret = array();
+        $ct = 0;
+        foreach ($this->glob($resource, false, $_, $ignoreErrors) as $resource => $info) {
+            ++$ct;
+            $ret[] = $this->doImport($resource, $type, $ignoreErrors, $sourceResource);
+        }
+
+        return $ct > 1 ? $ret : $ret[0] ?? null;
```

Comment:
This is ugly as `load` and `import` are not supposed to return anything, but the routing loaders do expect some return value (something that should probably be fixed at some point).